cargo-apk/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Unreleased
 
+- Added support for dynamic libraries loaded at runtime.
```

Comment:
This doesn't add support for loading dynamic libs at runtime per-se (that's already supported), but rather _packaging them up in the APK_ to make this possible with libraries other than preexisting ones on the device. Can you word it something like that pending future changes to the path etc?